src/main/java/org/neo4j/gis/spatial/osm/OSMImporter.java

```diff
@@ -2131,6 +2132,15 @@ public static RoadDirection isOneway( Map<String, Object> wayProperties )
         }
         return RoadDirection.BOTH;
     }
+    
+    /**
+     * Legacy version of {@link #getRoadDirection(Map)}, for code using it under the old name.
+     * @param wayProperties the property map of the road
+     * @return same as {@link #getRoadDirection(Map)}
+     */
+    public static RoadDirection isOneway(Map<String, Object> wayProperties) {
+    	return getRoadDirection(wayProperties);
+    }
 
```

Comment:
Since this code is very likely only used internally, despite this being a public method, I have no objection removing this method. Also, all of Neo4j Spatial is alpha code, and changing the API is completely acceptable at this point. Should there be users that are relying on this we can either revert that decision, or support them changing to the new API.